templates/geoipupdate.timer.j2

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Periodic GeoIP update
+
+[Timer]
+OnCalendar={{ geoip2_geoipupdate_service_timer_on_calendar }}
+RandomizedDelaySec={{ geoip2_geoipupdate_service_timer_randomized_delay_sec }}
```

Comment:
Please see [my related comment in `README.md`](https://github.com/cisagov/ansible-role-geoip2/pull/51#discussion_r1664224157):
```suggestion
RandomizedDelaySec={{ geoip2_geoipupdate_service_timer_randomized_delay }}
```